src/modules/px4iofirmware/mixer.cpp

```diff
@@ -301,6 +310,17 @@ mixer_tick(void)
 		for (unsigned i = 0; i < PX4IO_SERVO_COUNT; i++) {
 			r_page_actuators[i] = FLOAT_TO_REG(outputs[i]);
 		}
+
+
+		if (mixed  && new_fmu_data) {
+			new_fmu_data = false;
+
+			/* Trigger all timer's channels in Oneshot mode to fire
+			 * the oneshots with updated values.
+			 */
+
+			up_pwm_update();
```

Comment:
This means, oneshot is off when there is nothing from the FMU. That should work.